client/lib/site/utils.js

```diff
@@ -172,5 +172,18 @@ export default {
 
 	isJetpack( site ) {
 		return site && site.jetpack;
+	},
+
+	/**
+	 * Returns the supplied URL without the initial http(s).
+	 * @param  {String}  url The URL to remove http(s) from
+	 * @return {?String}     URL without the initial http(s)
+	 */
+	withoutHttp( url ) {
+		if ( ! url ) {
+			return null;
+		}
+
+		return url.replace( /^https?:\/\//, '' );
```

Comment:
If our regexes are static, I prefer defining them outside of the calling function so they don't need to be created each time.